tests/test/then.js

```diff
@@ -148,13 +148,14 @@ describe("Spooky provides Casper's then* functions", function () {
             function (done) {
                 context.spooky.start(FIXTURE_URL + '/1.html');
 
-                context.spooky.thenEvaluate(function (foo) {
+                context.spooky.thenEvaluate(function (foo, bar) {
                     window.__foo = foo;
-                }, { foo: 'bar' });
+                    window.__bar = bar;
+                }, 'foo', 'bar');
```

Comment:
This removes the test for the old style of passing args. I don't know if that's what we want to do.